python/tvm/relay/frontend/onnx.py

```diff
@@ -1391,7 +1391,7 @@ def _impl_v1(cls, inputs, attr, params):
         dtype = input0_state.checked_type.dtype
         # Y = alpha * A * B + beta * C
         alpha = float(attr.get("alpha", 1.0))
-        beta = float(attr.get("beta", 1.0))
+        beta = float(attr.get("beta"))
```

Comment:
Though the change on L1409 might not be needed since if beta == 1, it can be removed with constant folding.

Reply:
Constant folding doesn't work when beta is multiplying an output of qnn ops, since we cannot fold over them. The model in https://github.com/apache/tvm/issues/13545 has `multiply(1f, dequantize(bias)` after `dense`, which was also causing some issues.

Reply:
Moved `float(beta)` to the `else` block of `if beta is None`.

Reply:
Actually the whole purpose of this change was to avoid multiplying by 1.0, since `multiply(1f, dequantize(bias)` would be converted to `qnn.mul(quantize(1), bias)`. So I restored the original code cc @Icemist